src/uct/cuda/cuda_copy/cuda_copy_iface.h

```diff
@@ -11,6 +11,12 @@
 #include <pthread.h>
 
 
+#define UCT_CUDA_COPY_IFACE_DEFAULT_BANDWIDTH (10000.0 * UCS_MBYTE)
+
+
+#define UCT_CUDA_COPY_IFACE_OVERHEAD          (0)
```

Comment:
IMO can remove these

Reply:
Are you sure? Notice that it is used in 2 places.
If I replace them with directly using "0" instead, and the overhead ever changes to a positive value in the future - we would need to remember to modify it in **2 places**, otherwise it would be inconsistent.

Reply:
ok, let's keep as is then